build.gradle

```diff
@@ -20,7 +20,7 @@ configurations {
 
 dependencyManagement {
     imports {
-        mavenBom 'org.testcontainers:testcontainers-bom:1.9.0'
+        mavenBom 'org.testcontainers:testcontainers-bom:1.10.7'
```

Comment:
why not 1.11.2? :D

Reply:
Why get it right when I can hilariously get it wrong over and over again?! :D

Thank you - fixed :)